core/src/main/java/org/elasticsearch/index/query/InnerHitBuilder.java

```diff
@@ -64,6 +59,7 @@
     public static final ParseField NAME_FIELD = new ParseField("name");
     public static final ParseField NESTED_PATH_FIELD = new ParseField("path");
     public static final ParseField PARENT_CHILD_TYPE_FIELD = new ParseField("type");
```

Comment:
It is used in the `build(...)` method to decide what sub search context implementation to build.